client/src/fabric/FabricConnection.ts

```diff
@@ -197,4 +203,8 @@ export abstract class FabricConnection {
     private getAllPeers(): Array<Client.Peer> {
         return this.gateway.getClient().getPeersForOrg();
     }
+
+    private getClientTimeout(): number {
```

Comment:
This shouldn't be done in here, it should be in the connect command and you pass the timeout through

Reply:
Removed this function and now pass in the timeout value in the connect command (when calling connect)